src/extension/common/components/copyButton.re

```diff
@@ -1,4 +1,10 @@
-let createElement ::label="copy" ::text ::onCopy ::style=? children::_ () =>
-  <CopyToClipboard text onCopy>
-    <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
-  </CopyToClipboard>;
+
+let component = ReasonReact.statelessComponent "CopyButton";
+let make ::label="copy" ::text ::onCopy ::style=? _ => {
+  ...component,
+
+  render: fun _ _ =>
+    <CopyToClipboard text onCopy>
+      <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
+    </CopyToClipboard>
+};
```

Comment:
Can you still write this as a statelessFunctional component that just returns JSX? Or will it fail?

```reason
let make ::label="copy" ::text ::onCopy ::style=? _ => 
  <CopyToClipboard text onCopy>
    <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
  </CopyToClipboard>;
```

cc @chenglou @cristianoc - if not that's quite a bummer.